recurly/base_client.py

```diff
@@ -61,7 +61,18 @@ def _make_request(self, method, path, body, params):
         except socket.error as e:
             raise NetworkError(e)
 
+    def _validate_path_parameters(self, args):
+        """Checks that path parameters are valid"""
+        # Check that parameters are valid types
+        if any(type(arg) not in [str, int, float] for arg in args):
+            raise ApiError("Invalid parameter type", None)
+
+        # Check that string parameters are not empty
+        if any(isinstance(arg, str) and not bool(arg.strip()) for arg in args):
+            raise ApiError("Parameters cannot be empty strings", None)
```

Comment:
I'm not sure if #393 will fix this, but even with this change, if you pass in `"/"` to `get_account()`, it will also return the list of accounts. Are there any other validations we can add to make sure it catches that one?

Reply:
#393 will address that issue:

```shell
Started GET "/accounts/%2F" for 172.19.0.1 at 2020-04-27 19:35:27.860 +0000
...
Rendering error: error_type=not_found error_message="Couldn't find Account with id = /"

```